### PR TITLE
Release v.1.0.0

### PR DESCRIPTION
**Pull request by YuiKasagi**
Release branch for version 1.0.0

#### Status
- Pipeline works both for the Y/J and H bands.
- The spectral variability between multiple frames of the same object's and observed on the same night was examined. I found that **they behave more stably with PyIRD** than with the conventional pipeline (with IRAF), mainly due to the improvement of the readout noise reduction.
- **Notably, readout noise removal for REACH data works better with PyIRD.**
- (These results will be shown in my PhD thesis.)

### Updates (v0.3.0 -> 1.0.0)
- Readout noise model reviewed.
- The spectrum extraction steps reviewed. eg.) the function `apext_flatfield` is implemented. Now that we implemented all the same reduction steps as the process with IRAF. 
- Minor adjustments for the continuum fitting, the Y/J band reduction.
- Add new class `Stream1D` for further reduction of 1D spectra (e.g., remove fringe, sky emission mask, etc...), while the functions have been still under development.

### CODE REVIEW

File: examples/python/REACH_stream.py [hunk 1]
```diff
@@ -6,90 +6,106 @@
...
-if flat.band=='h':
+anadir = basedir/'reduc'
+if band=='h':
     rawtag='IRDAD000'
-elif flat.band=='y':
+    ignore_orders=None
+elif band=='y':
     rawtag='IRDBD000'
+    ignore_orders = list(np.arange(1,32))+list([50,51])
+thar=irdstream.Stream2D("thar",datadir,anadir,fitsid=list(range(53347,53410,2)),inst=inst,rawtag=rawtag)#
```
Comment (kemasuda) [inline examples/python/REACH_stream.py:70]: thar.band = band?

**Review by kemasuda — APPROVED** [review]
I re-reduced the MMF/REACH spectra with v1.0.0 and it seems to be working well!

File: examples/python/REACH_stream.py (hunk 1 above)
Comment (YuiKasagi) [reply examples/python/REACH_stream.py:70]: Thank you for pointing this out!
It looks `rawtag` was mistakenly set for this data set, so the band had been set incorrectly. I fixed this by commenting out `rawtag` in Stream2D.
(We need to set `rawtag` for data starting from `IRDAD` or `IRDBD`, but do not have to data starting from `IRDA`.)

**Review by HajimeKawahara — APPROVED** [review]
Thanks!

**Pull request merged**